src/Resources/templates/get.php.twig

```diff
@@ -18,10 +18,14 @@
  {% if property.collection -%}
  * @return {{ property.fullyQualifiedType }}[]|ImmutableCollection
  {% else -%}
- * @return {{ (property.isComplexType ? property.typeHint : property.type) | phptype }}{{ not property.willGenerateStrict ? '|null' }}
+ * @return {{ (property.isComplexType ? property.typeHint : property.type) | phptype }}{{ property.isNullable or not property.willGenerateStrict ? '|null' }}
  {% endif -%}
  */
-{{ property.getGetVisibility() }} function {{ getter }}()
+{{ property.getGetVisibility() }} function {{ getter }}(){% if property.type != 'resource' and property.type != 'object' %}: {% if property.collection -%} iterable
```

Comment:
Would be nicer to write this on multiple lines. I suspect this was done because otherwise Twig will add some new lines, but they have some features to control the white space: https://twig.symfony.com/doc/2.x/templates.html#whitespace-control